.changeset/changelog.js

```diff
@@ -0,0 +1,58 @@
+// Slightly modified version of
+// https://github.com/atlassian/changesets/blob/master/packages/changelog-github/src/index.ts
+// changing the release line formatting
+const { getInfo } = require('@changesets/get-github-info');
+
+const repo = 'seek-oss/braid-design-system';
+
+const changelogFunctions = {
+  getDependencyReleaseLine: async (changesets, dependenciesUpdated) => {
+    if (dependenciesUpdated.length === 0) return '';
+
+    const changesetLink = `- Updated dependencies [${(
+      await Promise.all(
+        changesets.map(async cs => {
+          if (cs.commit) {
+            let { links } = await getInfo({
+              repo,
+              commit: cs.commit,
+            });
+            return links.commit;
+          }
+        }),
+      )
+    )
+      .filter(_ => _)
+      .join(', ')}]:`;
+
+    const updatedDepenenciesList = dependenciesUpdated.map(
+      dependency => `  - ${dependency.name}@${dependency.newVersion}`,
+    );
+
+    return [changesetLink, ...updatedDepenenciesList].join('\n');
+  },
+  getReleaseLine: async changeset => {
+    const [firstLine, ...futureLines] = changeset.summary
+      .split('\n')
+      .map(l => l.trimRight());
+
+    if (changeset.commit) {
+      let { links } = await getInfo({
+        repo,
+        commit: changeset.commit,
+      });
+
+      const versionInfo = `${changeset.commit} ${
+        links.pull === null ? '' : links.pull
+      }`;
+
+      const summary = `- ${firstLine}: ${versionInfo}`;
```

Comment:
I think now that were formatting this we can design it a bit more. I dont feel we need the commit if we have a link to the PR, so would it make more sense to do something like:
```ts
const summary = `- ${firstLine}${links.pull ? ` (${links.pull})` : ''}` 
```